File: app/src/main/java/hv/anwb/nl/flappywacht/MainActivity.java
package hv.anwb.nl.flappywacht;

import android.graphics.Point;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Display;

import hv.anwb.nl.flappywacht.easteregg.GameView;

public class MainActivity extends AppCompatActivity {

    private GameView gameView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Display display = getWindowManager().getDefaultDisplay();
        Point size = new Point();
        display.getSize(size);
        gameView = new GameView(this, size.x, size.y);
        setContentView(gameView);
    }

    @Override
    protected void onPause() {
        super.onPause();
        gameView.pause();
    }

    @Override
    protected void onResume() {
        super.onResume();
        gameView.resume();
    }
}

File: app/src/main/java/hv/anwb/nl/flappywacht/MainActivity.kt
package hv.anwb.nl.flappywacht

import android.graphics.Point
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import hv.anwb.nl.flappywacht.easteregg.GameView

class MainActivity : AppCompatActivity() {

    private lateinit var gameView: GameView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val size = Point()
        windowManager.defaultDisplay.getSize(size)
        gameView = GameView(this, size.x, size.y)
        setContentView(gameView)
    }

    override fun onPause() {
        super.onPause()
        gameView.pause()
    }

    override fun onResume() {
        super.onResume()
        gameView.resume()
    }
}